packages/vite/schemas/manifest.schema.json

```
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Vite Manifest",
+  "type": "object",
+  "patternProperties": {
+    "^.+$": {
+      "type": "object",
+      "properties": {
+        "src": {
+          "type": "string"
+        },
+        "file": {
+          "type": "string"
+        },
+        "css": {
+          "type": "array",
+          "items": {
+            "type": "string"
```

Perhaps we could add a `description`:
```suggestion
            "type": "string",
            "description": "A CSS filename, relative to the outDir."
```